src/edu/washington/escience/myria/operator/SymmetricHashJoin.java

```diff
@@ -32,279 +27,93 @@
   /** Required for Java serialization. */
   private static final long serialVersionUID = 1L;
 
-  /**
-   * The names of the output columns.
-   */
+  /** The names of the output columns. */
   private final ImmutableList<String> outputColumns;
-
-  /**
-   * The column indices for comparing of child 1.
-   */
+  /** The column indices for comparing of the left child. */
   private final int[] leftCompareIndx;
```

Comment:
After another though I think `leftCompareColumns` sounds better, it also matches `leftAnswerColumns`.